blog/en/5_discovering-the-nike-ecommerce.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Enginyeria inversa del ecommerce de Nike amb només el navegador
```

Comment:
IMO posaria el contingut del títol en format 'slugified' a la URL. Per exemple: enginyeria-inversa-del-ecommerce-de-nike, en comptes de `discovering-the-nike-ecommerce`.

Crec que el primer es millor també, ja que el títol indica clarament que és un blog tècnic, en canvi "Descobrint el ecommerce de Nike" pot significar moltes coses.